cmd/loop/loopout.go

```diff
@@ -198,7 +198,13 @@ func loopOut(ctx *cli.Context) error {
 		)
 	}
 
-	// Override the old maxMinerFee.
+	// Check that the maxMinerFee is sane.
+	if maxMinerFee < limits.minMinerFee {
+		return fmt.Errorf(
```

Comment:
nit: for `fmt.Errorf()` and all `log.Xxxf()` we usually leave the format string on the same line to make it a bit more compact. Those format strings are the only exception to the "place too long arguments on a new line" rule.

Reply:
Good to know, now changed.